sample-lnd.conf

```diff
@@ -834,6 +834,13 @@ litecoin.node=ltcd
 ; Allow outbound and inbound connections to be routed through Tor
 ; tor.active=true
 
+; Allow the node to connect to non-onion services directly via clearnet. This
+; allows the node operator to use direct connections to peers not running behind
+; Tor, thus allowing lower latency and better connection stability.
+; WARNING: This option will reveal the source IP address of the node, and should
+; be used only if privacy is not a concern.
+; tor.directconnections=true
```

Comment:
Solid helper text here, I can't help but think maybe we should modify the _name_ of this new option though... 🤔

Don't have any solid suggestions here, but will circle back if I can think of something.

Reply:
I would rename every occurrence of "direct" with "clearnet". (except for "directly via clearnet") So the option would be `tor.clearnetconnections`